src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/Configurations/Configuration.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// This is a generated file from the original deployment recipe. It contains properties for
+// all of the settings defined in the recipe file. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// This class is marked as a partial class. If you add new settings to the recipe file, those settings should be
+// added to partial versions of this class outside of the Generated folder for example in the Configuration folder.
+
+namespace AspNetAppElasticBeanstalkLinux.Configurations
+{
+    public partial class Configuration
+    {
+        /// <summary>
+        /// The Identity and Access Management Role that provides AWS credentials to the application to access AWS services
+        /// </summary>
+        public IAMRoleConfiguration ApplicationIAMRole { get; set; }
+
+        /// <summary>
+        /// The type of environment for the Elastic Beanstalk application.
+        /// </summary>
+        public string EnvironmentType { get; set; } = Recipe.ENVIRONMENTTYPE_SINGLEINSTANCE;
+
+        /// <summary>
+        /// The EC2 instance type used for the EC2 instances created for the environment.
+        /// </summary>
+        public string InstanceType { get; set; }
+
+        /// <summary>
+        /// The Elastic Beanstalk environment name.
+        /// </summary>
+        public string EnvironmentName { get; set; }
+
+        /// <summary>
+        /// The Elastic Beanstalk application.
+        /// </summary>
+        public BeanstalkApplicationConfiguration BeanstalkApplication { get; set; }
+
+        /// <summary>
+        /// The name of an Elastic Beanstalk solution stack (platform version) to use with the environment.
+        /// </summary>
+        public string ElasticBeanstalkPlatformArn { get; set; }
+
+        /// <summary>
+        /// The type of load balancer for your environment.
+        /// </summary>
+        public string LoadBalancerType { get; set; } = "application";
```

Comment:
can you create a constant similar to Recipe.ENVIRONMENTTYPE_SINGLEINSTANCE ?

Reply:
Done